db/seeds.rb

```diff
@@ -278,46 +278,61 @@
-Request.create!(user_id: user_seeker_5.id, category_id: category_mobile.id)
-Request.create!(user_id: user_seeker_7.id, category_id: category_printer.id)
-Request.create!(user_id: user_seeker_3.id, category_id: category_desktop.id)
-Request.create!(user_id: user_seeker_3.id, category_id: category_monitor.id)
-Request.create!(user_id: user_seeker_6.id, category_id: category_printer.id)
-Request.create!(user_id: user_seeker_4.id, category_id: category_camera.id)
-Request.create!(user_id: user_seeker_8.id, category_id: category_laptop.id)
-Request.create!(user_id: user_seeker_6.id, category_id: category_tablet.id, status: "Completed")
+Request.create!(user_id: user_seeker_1.id, category_id: category_laptop.id, description: "Amal needs a laptop to complete her university degree.")
+Request.create!(user_id: user_seeker_2.id, category_id: category_laptop.id, description: "Saloo needs a laptop to study Social Work.", status: "Completed")
+Request.create!(user_id: user_seeker_5.id, category_id: category_mobile.id, description: "Noor needs a mobile phone to start her own catering business.")
+Request.create!(user_id: user_seeker_7.id, category_id: category_printer.id, description: "Sulaiman needs a desktop computer and a printer to continue his job as an engineer")
+Request.create!(user_id: user_seeker_3.id, category_id: category_desktop.id, description: "Bob needs a desktop and monitor to continue his journey to be a software engineer.")
+Request.create!(user_id: user_seeker_3.id, category_id: category_monitor.id, description: "Bob needs a desktop and monitor to continue his journey to be a software engineer.")
+Request.create!(user_id: user_seeker_4.id, category_id: category_camera.id, description: "Abebe needs a camera so that he can continue his career as a photographer.")
+Request.create!(user_id: user_seeker_8.id, category_id: category_laptop.id, description: "Wafaa needs a laptop to finish her final year of university and find a job.")
+Request.create!(user_id: user_seeker_6.id, category_id: category_tablet.id, description: "Zaida needs a laptop to study and complete her final year of university.", status: "Completed")
 
 #bookings
 Booking.create!(charity_id: charity2.id, 
-                user_id: user_seeker_1.id, 
+                user_id: user_seeker_2.id, 
                 product_id: product_dell_laptop.id, 
-                status: "Declined"
+                status: "Accepted"
                 )
-Booking.create!(charity_id: charity2.id, 
-                user_id: user_seeker_2.id, 
+Booking.create!(charity_id: charity1.id, 
+                user_id: user_seeker_1.id, 
                 product_id: product_macbook.id, 
-                status: "Accepted"
+                status: "Declined"
                 )
-Booking.create!(charity_id: charity2.id, 
-                user_id: user_seeker_5.id, 
+Booking.create!(charity_id: charity1.id, 
+                user_id: user_seeker_8.id, 
                 product_id: product_mobile.id, 
```

Comment:
I think this needs to be a laptop as this seeker requests a laptop

Reply:
Thanks @RyanofWoods , missed that